src/player_activity.h

```diff
@@ -261,53 +167,19 @@ class player_activity
          * Most of those are quite self-explanatory by the name
         */
 
-
-        inline void init_all_moves( Character &who ) {
-            if( actor ) {
-                actor->recalc_all_moves( *this, who );
-            } else {
-                recalc_all_moves( who );
-            }
-        }
+        void init_all_moves( Character &who );
 
         //Calculates speed factors that may change every turn
-        void calc_moves( const Character &who );
-
-        //Calculates all factors
-        void recalc_all_moves( Character &who );
-        void recalc_all_moves( Character &who, activity_reqs_adapter &reqs );
-
-        //Fills bench var
-        void find_best_bench( const tripoint &pos );
-        float calc_bench_factor( const Character &who ) const;
-
-        float calc_light_factor( const Character &who ) const;
-        std::vector<std::pair<character_stat, float>> calc_stats_factors( const Character &who ) const;
+        inline void calc_moves( const Character &who ) {
```

Comment:
```suggestion
        void calc_moves( const Character &who ) {
```
No need to specify it as inline here, it's implicitly inline